packages/lodestar/src/db/beacon.ts

```diff
@@ -48,7 +48,7 @@ export class BeaconDb extends DatabaseService implements IBeaconDb {
     this.badBlock = new BadBlockRepository(this.config, this.db);
     this.block = new BlockRepository(this.config, this.db);
     this.pendingBlock = new PendingBlockRepository(this.config, this.db);
-    this.seenAttestationCache = new SeenAttestationCache(5000);
+    this.seenAttestationCache = new SeenAttestationCache(this.config, 2048);
```

Comment:
curious why this is decreased?

Reply:
currently it's used for 2 caches: (committee) Attestation and AttestationAndProof but this PR changes to cache them separately so mathematically it should be `5000 / 2 = 2500`, but I changed it to `2048`